end_to_end/signal_handler_test.go

```diff
@@ -11,6 +13,29 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func CreateCombinedOutput(cmd *exec.Cmd) *bufio.Scanner {
```

Comment:
I suggest renaming to createCombinedOutput() to eliminate unneeded symbol export.